.npmrc

```diff
@@ -1 +1,2 @@
 engine-strict=true
+node-options="--openssl-legacy-provider"
```

Comment:
Hey @dantovbein 
Is it possible to update our dependencies to ensure they’re compatible with OpenSSL v3 instead of using `--openssl-legacy-provider`?

Reply:
To be honest, unfortunately nope.

On the one hand, we cannot manage all the dependencies we have integrated since many of them are coming from third parties. On the other hand, we all use different OS archs, so it's fine if we set this variable before install and make sure we are all in the same road.